servers/src/main/java/tachyon/master/file/meta/TTLBucket.java

```diff
@@ -80,4 +82,27 @@ public int compareTo(TTLBucket ttlBucket) {
     }
     return 1;
   }
+
+  /**
+   * Compares to a specific object.
+   *
+   * @param object the object to compare
+   * @return true if object is also {@link TTLBucket} and represents
+   * the same TTLIntervalStartTime and files
+   */
+  @Override
+  public boolean equals(Object object) {
+    if (object == null || getClass() != object.getClass()) {
+      return false;
+    }
+
+    TTLBucket that = (TTLBucket) object;
+    return Objects.equal(mTTLIntervalStartTimeMs, that.mTTLIntervalStartTimeMs)
```

Comment:
TTLBucket overrides `compareTo` method, to compare `mTTLIntervalStartTimeMs` only.
Shall we make `equals` method be consistent with it?